sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/JsonHelper.java

```diff
@@ -0,0 +1,93 @@
+package com.azure.identity.implementation;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.microsoft.aad.msal4j.AuthenticationErrorCode;
+import com.microsoft.aad.msal4j.ClaimsRequest;
+import com.microsoft.aad.msal4j.MsalClientException;
+import com.microsoft.aad.msal4j.RequestedClaimAdditionalInfo;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.Set;
+
+public class JsonHelper {
```

Comment:
Remove it, if not shipping In memory cache.

Reply:
We can remove this.